profiles/serializers.py

```diff
@@ -102,14 +106,13 @@ class Meta:
 
 class ProfileSerializer(ModelSerializer):
     """Serializer for Profile objects"""
-    pretty_printed_student_id = SerializerMethodField()
+    username = SerializerMethodField()
     work_history = EmploymentSerializer(many=True)
     education = EducationSerializer(many=True)
 
-    def get_pretty_printed_student_id(self, obj):
```

Comment:
I'm guessing this was unnecessary, the serializer just needs the name in the fields list?